modules/mtl/src/test/scala/natchez/mtl/http4s/syntax/EntryPointOpsSuite.scala

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) 2021 by Rob Norris
+// This software is licensed under the MIT License (MIT).
+// For more information see LICENSE or https://opensource.org/licenses/MIT
+
+package natchez.mtl.http4s.syntax
+
+import cats.arrow.FunctionK
+import cats.effect.{IO, IOLocal, MonadCancelThrow}
+import cats.mtl.Local
+import cats.syntax.all.*
+import cats.{Applicative, Monoid}
+import munit.{Location, ScalaCheckEffectSuite, TestOptions}
+import natchez.Span.Options.Defaults
+import natchez.Span.Options.SpanCreationPolicy.{Coalesce, Default, Suppress}
+import natchez.Span.SpanKind.Server
+import natchez.http4s.InMemorySuite
+import natchez.http4s.syntax.kernel.*
+import natchez.{EntryPoint, InMemory, Kernel, Span, Trace}
+import natchez.mtl.*
+import org.http4s.dsl.Http4sDsl
+import org.http4s.syntax.all.*
+import org.http4s.{Header, Headers, HttpApp, HttpRoutes, Method, Request}
+import org.scalacheck.effect.PropF
+
+class EntryPointOpsSuite
+  extends InMemorySuite
+    with ScalaCheckEffectSuite {
+
+  // TODO remove if https://github.com/typelevel/natchez/pull/1071 is merged
```

Comment:
https://github.com/typelevel/natchez/pull/1071 was merged, so I'll remove these instances from this PR once the updated natchez-testkit is released.